pkg/lib/cockpit/_internal/transport.ts

```diff
@@ -198,7 +198,7 @@ class Transport extends EventEmitter<{ ready(): void }> {
                 transport_debug("recv control:", control);
```

Comment:
It would be clearer to localize the declaration here as `const control =`. It's just confusing and unnecessarily nullable that way.

Reply:
I don't see how it can be const, it is also used way below on line [209](https://github.com/cockpit-project/cockpit/pull/21023/files#diff-eb9dca6be2bf9e6b23dd2aaaffbad08cb665f213392f5b645aa8ea4105f67fdcR20).

Reply:
Ah right, sorry.